modules/dreamview/frontend/src/renderer/index.js

```diff
@@ -493,6 +498,14 @@ class Renderer {
         const names = intersects.map(intersect => intersect.object.name);
         return names;
     }
+
+    addTrafficControl(type, elements) {
```

Comment:
TrafficControl is still part of the map and its add/remove is controlled by map. What if we move this TrafficControl object to map/tileground?